fineract-provider/src/main/java/org/apache/fineract/portfolio/client/command/ClientIdentifierCommand.java

```diff
@@ -50,18 +57,29 @@ public String getDocumentKey() {
         return this.documentKey;
     }
 
+    public String getdocumentIssueCountry() {
+        return this.documentIssueCountry;
+    }
+
     public String getDescription() {
         return this.description;
     }
 
-    public void validateForCreate() {
+    public void validateForCreate(final CodeValue documentType, final JsonCommand command) {
         final List<ApiParameterError> dataValidationErrors = new ArrayList<>();
 
         final DataValidatorBuilder baseDataValidator = new DataValidatorBuilder(dataValidationErrors).resource("clientIdentifier");
 
         baseDataValidator.reset().parameter("documentTypeId").value(this.documentTypeId).notNull().integerGreaterThanZero();
         baseDataValidator.reset().parameter("documentKey").value(this.documentKey).notBlank();
 
+        if (documentType.label().equals(ClientApiConstants.clientIdentifierPassportParamName.toString())) {
```

Comment:
Is there a need to actually call the toString() on that constant? It would also even better if we could switch the order of the equality check to be null-safe.
Thoughts?

Reply:
Addressed,